src/Pact/Types/Native.hs

```diff
@@ -42,12 +42,16 @@ isSpecialForm = (`M.lookup` sfLookup)
 
 
 -- | Native function with un-reduced arguments that computes gas.
+--   Opertations with cost dependent on amount of computation performed.
 type NativeFun e = FunApp -> [Term Ref] -> Eval e (Gas,Term Name)
 
 -- | Native function with reduced arguments, initial gas pre-compute that computes final gas.
+--   Gas = Cost of Opertation * How Many Times Operation Occurs
```

Comment:
This comment is misleading. Whatever gets added on is up to the implementation of the native.